### PR TITLE
add nfo files for original one piece wano episodes

### PR DESCRIPTION
**Pull request by TheIceCreamTroll**
## Describe your changes
Added nfo files for the original One Piece episodes needed for Wano

## Checklist for submitting new .nfo files
- [x] I have removed the old .nfo files that are being replaced
- [x] I have validated that the new .nfo file works correctly in Plex
- [x] I have updated the README, fixing the list of current missing episodes
- [x] I have modified exceptions.json, removing any exceptions no longer required

### CODE REVIEW

**Comment by cawnj** [comment]
Nice work! At first glance it looks good to me, but can you add some screenshots of your library in Plex showing a couple of the episodes added and having the correct metadata?

**Comment by TheIceCreamTroll** [comment]
> Nice work! At first glance it looks good to me, but can you add some screenshots of your library in Plex showing a couple of the episodes added and having the correct metadata?

image

image

image

**Comment by cawnj** [comment]
Ok nice, thank you @TheIceCreamTroll and great job on getting it done so quick!
I'd say the change is good to go, but since it's such a big change, I'll let @SpykerNZ also review, just in case I've missed something here.
Thanks!!

File: README.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Current missing episodes:
 - Season 16: 171-195, 207
 - Season 18: 262-263
 - Season 24: 453-456
+- Season 34 - 968-1085
```
Comment (SpykerNZ) [inline README.md:71]: should follow same bullet point structure here for consistency: `Season 34: 968-1085`

**Review by SpykerNZ — APPROVED** [review]
Looks great! If you could touch up the one line in the README, then happy for it to be merged in 🙌 Thanks for your contribution 🎉

File: README.md (hunk 1 above)
Comment (TheIceCreamTroll) [reply README.md]: Whoops, nice catch! Fixed

**Comment by TheIceCreamTroll** [comment]
README fixed

**Pull request merged**